frontend/viewer/src/lib/entry-editor/NewEntryDialog.svelte

```diff
@@ -24,6 +24,7 @@
   async function createEntry(e: Event, closeDialog: () => void) {
     e.preventDefault();
     loading = true;
+    console.debug('Creating entry', entry);
```

Comment:
forgot this

Reply:
I'd like to leave this.
I've started leaving `console.debug` statements around and find them to be handy, because they're invisible in the console unless you go looking for them.
![image](https://github.com/user-attachments/assets/0b98e3f2-b48b-48f9-b6c6-ce185dc9b21a)